deps/BUILD.bazel

```diff
@@ -389,8 +389,16 @@ proto_dependency(
 proto_dependency(
     name = "build_bazel_rules_nodejs",
     repository_rule = "http_archive",
-    sha256 = "4501158976b9da216295ac65d872b1be51e3eeb805273e68c516d2eb36ae1fbb",
-    urls = ["https://github.com/bazelbuild/rules_nodejs/releases/download/4.4.1/rules_nodejs-4.4.1.tar.gz"],
+    sha256 = "dcc55f810142b6cf46a44d0180a5a7fb923c04a5061e2e8d8eb05ccccc60864b",
+    urls = ["https://github.com/bazelbuild/rules_nodejs/releases/download/5.8.0/rules_nodejs-5.8.0.tar.gz"],
+    deps = ["rules_nodejs"],
+)
+
+proto_dependency(
+    name = "rules_nodejs",
+    repository_rule = "http_archive",
+    sha256 = "08337d4fffc78f7fe648a93be12ea2fc4e8eb9795a4e6aa48595b66b34555626",
+    urls = ["https://github.com/bazelbuild/rules_nodejs/releases/download/5.8.0/rules_nodejs-core-5.8.0.tar.gz"],
 )
```

Comment:
is rules_nodejs a dependency for ts ?

Reply:
I think so. I can't tell if it's been missing all along or if changes in `build_bazel_rules_node_js`/`rules_nodejs` between version 4.x and 5.x necessitate adding it to the ts-proto deps. (I believe `build_bazel_rules_node_js` and `rules_nodejs` were the same thing in 4.x and were split in 5.x, hence this extra repository.)